daprdocs/content/en/operations/resiliency/policies.md

```diff
@@ -91,3 +93,41 @@ spec:
+
+> Note: Although you can override default values with more robust retries, you cannot override with lesser values than the provided default value, or completely remove default retries. This prevents unexpected downtime. 
+
+Below is a table that describes Dapr's default retries and the policy keywords to override them: 
+
+| Capability             | Override Keyword                 | Default Retry Behavior                                                                                                                                                               | Description                                                                                                                           |
+| ------------------     | -------------------------        | ------------------------------                                                                                                                                                       | -----------------------------------------------------------------------------------------------------------                           |
+| Service Invocation     | DaprBuiltInServiceRetries        | Per call retries are performed with a backoff interval of 1 second up to a threshold of 3 times.                                                                                     | Sidecar-to-sidecar requets (via a service invocation call) that fails and results in a gRPC code `Unavailable` or `Unauthenticated`   | 
+| Actors                 | DaprBuiltInActorRetries          | Per call retries are performed with a backoff interval of 1 second up to a threshold of 3 times.                                                                                     | Sidecar-to-sidecar requests (to a remote actor) that fails and results in a gRPC code `Unavailable` or `Unauthenticated`              |
+| Actor Reminders        | DaprBuiltInActorReminderRetries  | Per call retries are performed with an exponential backoff wtih an initial interval of 500ms and up to a maximum of 60s for a duration of 15m                                        | Requests that fail to persist an actor reminder to a state store                                                                      | 
+| Initialization Retries | DaprBuiltInInitializationRetries | Per call retries are performed 3 times wtih an exponential backoff with an initial interval of 500ms and for a duration of 10s                                                       | Failures when making a request to an application to retrieve a given spec. For example, failure to retrieve a Subscription spec       | 
+
+
+Example of overriding default retries for Service Invocation:
```

Comment:
I would describe what this example is supposed to show, so then the reader has an expectation.